scripts/build-es-shims-data/es-shims-features.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+module.exports = {
+  globalThis: "globalThis",
+  AggregateError: [],
+  "Array.from": "Array static methods / Array.from",
+  "Array.of": "Array static methods / Array.of",
+  "Array.prototype.entries":
+    "Array.prototype methods / Array.prototype.entries",
+  "Array.prototype.every": "Array methods / Array.prototype.every",
+  "Array.prototype.find": "Array.prototype methods / Array.prototype.find",
+  "Array.prototype.findIndex":
+    "Array.prototype methods / Array.prototype.findIndex",
+  "Array.prototype.flat":
+    "Array.prototype.{flat, flatMap} / Array.prototype.flat",
+  "Array.prototype.flatMap":
+    "Array.prototype.{flat, flatMap} / Array.prototype.flatMap",
+  "Array.prototype.includes": "Array.prototype.includes",
+  "Array.prototype.indexOf": "Array methods / Array.prototype.indexOf",
+  "Array.prototype.keys": "Array.prototype methods / Array.prototype.keys",
+  "Array.prototype.lastIndexOf": "Array methods / Array.prototype.lastIndexOf",
+  "Array.prototype.map": "Array methods / Array.prototype.map",
+  "Array.prototype.reduce": "Array methods / Array.prototype.reduce",
+  "Array.prototype.reduceRight": "Array methods / Array.prototype.reduceRight",
+  "Array.prototype.some": "Array methods / Array.prototype.some",
+  "Array.prototype.values": "Array.prototype methods / Array.prototype.values",
+  "Function.prototype.name": {
+    features: [
+      'function "name" property / function statements',
+      'function "name" property / function expressions',
+    ],
+  },
+  "Number.isNaN": "Number properties / Number.isNaN",
+  "Object.assign": "Object static methods / Object.assign",
+  "Object.is": "Object static methods / Object.is",
+  "Object.entries": "Object static methods / Object.entries",
+  "Object.fromEntries": "Object.fromEntries",
+  "Object.getOwnPropertyDescriptors":
+    "Object static methods / Object.getOwnPropertyDescriptors",
+  "Object.values": "Object static methods / Object.values",
+  "Promise.allSettled": [],
+  "Promise.any": [],
+  "Promise.try": [],
+  "Promise.prototype.finally": "Promise.prototype.finally",
+  "Reflect.ownKeys": "Reflect / Reflect.ownKeys",
+  "RegExp.prototype.flags":
+    "RegExp.prototype properties / RegExp.prototype.flags",
+  "String.fromCodePoint": "String static methods / String.fromCodePoint",
+  "String.prototype.codePointAt":
+    "String.prototype methods / String.prototype.codePointAt",
+  "String.prototype.endsWith":
+    "String.prototype methods / String.prototype.endsWith",
+  "String.prototype.includes":
+    "String.prototype methods / String.prototype.includes",
+  "String.prototype.matchAll": [],
+  "String.prototype.padEnd": "String padding / String.prototype.padEnd",
+  "String.prototype.padStart": "String padding / String.prototype.padStart",
+  "String.prototype.repeat":
+    "String.prototype methods / String.prototype.repeat",
+  "String.prototype.replaceAll": [],
```

Comment:
@pustovalov For your PRs, it should now be enough to add here the corresponding [`compat-table`](https://kangax.github.io/compat-table/es2016plus/) entry. It supports three formats:
- `"category"`
- `"category / subcategory"`
- ```js
  {
    features: [
      "category1 / subcategory1",
      "category2 / subcategory2"
    ]
  }
  ```

Then, you need to run `yarn build-es-shims-data`.